tests/unit_tests/experiment_plans/test_wait_for_robot_load_then_centre.py

```diff
@@ -44,3 +51,94 @@ def test_when_plan_run_then_centring_plan_run_with_expected_parameters(
 
     assert composite_passed == mock_composite
     assert isinstance(params_passed, PinCentreThenXrayCentreInternalParameters)
+
+
+def run_simulating_smargon_wait(
```

Comment:
This feels like a much better to test things compared to what we used to be doing!

Reply:
I agree, thank you @rtuck99 for implementing it